docs/csharp/quick-starts/numbers-in-csharp.yml

```diff
@@ -0,0 +1,271 @@
+    more complicated expression with doubles:
+
+    ```csharp
+    double a = 19;
+    double b = 23;
+    double c = 8;
+    double d = (a  + b) / c;
+    Console.WriteLine(d);
+    ```
+
+    The range of a double value is much greater than integer values. Try the following
+    code in the interactive window:
+
+    ```csharp
+    double max = double.MaxValue;
+    double min = double.MinValue;
+    Console.WriteLine("The range of double is " + min + " to " + max);
+    ```
+
+    These values are printed out to represent scientific notation. The number to
+    the left of the `E` is the significant. The number to the right is the exponent,
+    as a power of 10. Internally, these numbers are represented similarly, but in 
+    a binary form. One part represents the significant binary digits.
+    Another part represents where to place the binary point (analogous to a decimal point) relative to
+    those significant digits.
+
+    Just like decimal numbers in math, doubles in C# can have rounding errors. Try this code:
+
+    ```csharp
+    double third = 1.0 / 3.0;
+    Console.WriteLine(third);
+    ```
+    
+    You know that `0.3` repeating is not exactly the same as `1/3`.
+
+        ***Challenge***
+
+    Try other calculations with large numbers, small numbers, multiplication
+    and division using the `double` type.  Try more complicated calculations.
+
+
+- title: Work with fixed point types
+  durationInMinutes: 4
+  content: |
+    You've seen the basic numeric types in C#: integers and doubles.  There is one
+    other type to learn: the `decimal` type. The `decimal` type has a smaller
+    range, but greater precision than `double`. The term **fixed point** means
+    that the decimal point (or binary point) does not move. Let's take a look:
+
+    ```csharp
+    decimal min = decimal.MinValue;
+    decimal max = decimal.MaxValue;
+    Console.WriteLine("the range of the decimal type is " + min + " to " + max);
+    ```
+
+    Notice that the range is smaller than the `double` type. You can see the greater
+    precision with the decimal type by trying the following code:
+
+    ```csharp
+    double a = 1.0;
+    double b = 3.0;
+    Console.WriteLine(a / b);
+
+    decimal c = 1.0M;
+    decimal d = 3.0M;
+    Console.WriteLine(c / d);
+    ```
+
+    The `M` suffix on the numbers is how you indicate that a constant should use the
+    `decimal` type.
+
+    Notice that the math using the decimal type has more digits to the right
+    of the decimal point. 
+
+    ***Challenge***
+
+    Now that you've seen the different numeric types, write code that calculates
+    the area of a circle whose radius is 2.50 inches. Remember that the area of a circle
+    is the radius squared multiplied by PI. One hint: C# contains a constant
+    for PI: `Math.PI` that you can use for that value. 
+
+- title: Complete challenge
+  durationInMinutes: 1
+  content: |
+    Did you come up with something like this?
```

Comment:
I'd separate the complete challenge from next steps as you did in the hello world tutorial. Otherwise, the window with the code disappears and I can't double check if I got my code right.